Compile issue on Rolling Kali 2017.02

Was seeing the following when following the instructions in the wiki:

> #make clean && make all
> cmdhfmf.c: In function ‘CmdHF14AMfCWipe’:
> cmdhfmf.c:1908:14: error: ‘TRUE’ undeclared (first use in this function)
>    wipeCard = TRUE;
>               ^~~~
> cmdhfmf.c:1908:14: note: each undeclared identifier is reported only once for each function it appears in
> Makefile:274: recipe for target 'obj/cmdhfmf.o' failed
> make[1]: *** [obj/cmdhfmf.o] Error 1
> make[1]: Leaving directory '/root/proxmark3/client'
> Makefile:33: recipe for target 'client/all' failed
> make: *** [client/all] Error 2

Modifying 'TRUE' to 'true' on line 1908 of cmdhfmf.c and recompiling seems to have resolved the issue.

--- a/client/cmdhfmf.c
+++ b/client/cmdhfmf.c
@@ -1905,7 +1905,7 @@ int CmdHF14AMfCWipe(const char *Cmd)
 	}
 
 	if (!wipeCard && !fillCard) 
-		wipeCard = TRUE;
+		wipeCard = true;
 
 	PrintAndLog("--blocks count:%2d wipe:%c fill:%c", numBlocks, (wipeCard)?'y':'n', (fillCard)?'y':'n');
 

